### PR TITLE
Editor: Add a full-brightness option

### PR DESCRIPTION
**Pull request by Hume2**
so the people can actually see how the dark levels look like.

### CODE REVIEW

**Comment by maxteufel** [comment]
Clarify the name of this. What does "Full Brightness" do? If you edit a level without darkness, you'd never notice a difference.

**Comment by Hume2** [comment]
@maxteufel Full brightness means that you have something like night-vision, the lighting just isn't rendered. It doesn't save into the level in any way. When there is no darkness in the level, it doesn't do anything.

**Comment by maxteufel** [comment]
@Hume2 yes. The option could be confusing though if one doesn't know what it does.

Also: make sure lightmap tiles are still rendered/displayed in some way.

File: src/video/drawing_context.cpp [hunk 1]
```diff
@@ -373,11 +375,13 @@ DrawingContext::do_drawing()
     handle_drawing_requests(lightmap_requests);
     lightmap.end_draw();
 
-    DrawingRequest* request = new(obst) DrawingRequest();
-    request->target = NORMAL;
-    request->type = DRAW_LIGHTMAP;
-    request->layer = LAYER_HUD - 1;
-    drawing_requests.push_back(request);
+    if (!full_brightness) {
+      DrawingRequest* request = new(obst) DrawingRequest();
```
Comment (tobbi) [inline src/video/drawing_context.cpp]: While we're at it:
auto request = ...

C'mon, show some respect for C++ 11. :P

**Comment by tobbi** [comment]
Anyone can think of a better name for this?

**Comment by maxteufel** [comment]
"Lighting" as a checkbox? @tobbi @Hume2

**Comment by qwertychouskie** [comment]
I like "Render lighting".

**Comment by tobbi** [comment]
Render lighting sounds good. What's a good German translation for this? "Lichteffekte rendern"?

**Comment by maxteufel** [comment]
@Hume2 Please change this to use "Render Lighting". If that's done, please remove your assignment.

File: src/video/drawing_context.cpp.orig [hunk 2]
```diff
@@ -0,0 +1,593 @@
+//  SuperTux
```
Comment (maxteufel) [inline src/video/drawing_context.cpp.orig]: I doubt you wanted to get this file into the repository.

**Pull request merged**